Solutions/Corvus.Testing.Azure.Cosmos.Tenancy/Corvus.Testing.Azure.Cosmos.Tenancy.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(EndjinProjectPropsPath)" Condition="$(EndjinProjectPropsPath) != ''" /> 
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Consider `<Nullable>enable</Nullable>`. It's easier to get it in there from the start than it is to retrofit.

Reply:
I agree. I was following "what I found" but not having nullable annoyed me.